src/main/java/org/openstreetmap/atlas/checks/validation/tag/MixedCaseNameCheck.java

```diff
@@ -0,0 +1,263 @@
+package org.openstreetmap.atlas.checks.validation.tag;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Map;
+import java.util.Optional;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import org.openstreetmap.atlas.checks.base.BaseCheck;
+import org.openstreetmap.atlas.checks.flag.CheckFlag;
+import org.openstreetmap.atlas.geography.atlas.items.AtlasObject;
+import org.openstreetmap.atlas.geography.atlas.items.LocationItem;
+import org.openstreetmap.atlas.geography.atlas.items.Relation;
+import org.openstreetmap.atlas.tags.ISOCountryTag;
+import org.openstreetmap.atlas.tags.annotations.validation.Validators;
+import org.openstreetmap.atlas.tags.names.NameTag;
+import org.openstreetmap.atlas.utilities.configuration.Configuration;
+
+/**
+ * This check flags objects with name tags that improperly use mixed cases.
+ *
+ * @author bbreithaupt
+ */
+public class MixedCaseNameCheck extends BaseCheck
+{
+
+    private static final long serialVersionUID = 7109483897229499466L;
+
+    private static final List<String> FALLBACK_INSTRUCTIONS = Arrays.asList(
+            "{0} {1,number,#} has (an) invalid mixed-case value(s) for the following tag(s): {2}.");
+    private static final List<String> CHECK_NAME_COUNTRIES_DEFAULT = Arrays.asList("AIA", "ATG",
```

Comment:
This should be unnecessary, all country based filtering is handled automatically in the base check.